misc/cf-support

```diff
@@ -182,6 +182,41 @@ if [ -f $WORKDIR/share/cf-support-nova-hub.sh ]; then
   source $WORKDIR/share/cf-support-nova-hub.sh
 fi
 
+echo "Analyzing CFEngine core dumps"
+_core_log="$tmpdir"/core-dump.log
+cf_core_files=$(find / -name 'core*' -type f -exec file {} \; 2>/dev/null | grep "core file" | grep "cf-" | cut -d' ' -f1 | sed 's/:$//')
```

Comment:
can i add a 2nd comment to the same line?

Note that if `gdb` is not installed, and user asks it not to be, then results of this `find /` are thrown away. Also, on some systems `find /` might take long time to run. Maybe it's better to rework this section into more user-friendly way:

1. first check if `gdb` is installed / available
2. then, ask _if_ user wants us to scan their system if search of core files
3. and only if user agrees, run the `find /` and `gdb`

Otherwise, good stuff!

Reply:
Sure. Makes sense. And if `--yes` obviously it will just go ahead and do both things, as in our testing setups.